### DISCUSSION TITLE
如何访问网络共享文件夹

### DISCUSSION THREAD

**Discussion by zjhappy1314** [Studio-42/elFinder#3652] [Q&A] [answered]
如题，我想访问局域网内的共享文件夹，配置path后提示：无效的服务器端配置。无可读的卷.
![配置如](https://github.com/Studio-42/elFinder/assets/18593232/670d4e95-a247-45e9-b4fe-a8f304bf26fb)
![提示](https://github.com/Studio-42/elFinder/assets/18593232/72693ee6-6dba-457e-b37b-77849d611a94)

**Comment by h3786010** [comment, marked as answer]
root value changed by getFullPath()
before: "\\\\IP\xyz"
after: "\IP\xyz""

Maybe can comment this line or override this function, But I'm not sure? @nao-pon 

https://github.com/Studio-42/elFinder/blob/66f497515ab2a3e867648cfb410089d1a8d6d53f/php/elFinderVolumeDriver.class.php#L6807-L6808

**Reply by zjhappy1314** [reply]
确实，注释掉这行就可以访问了。
